content/enterprise_influxdb/v1.9/administration/configure/config-data-nodes.md

```diff
@@ -9,6 +9,7 @@ menu:
     parent: Configure
 aliases:
   - /enterprise_influxdb/v1.9/administration/config-data-nodes/
+  - /enterprise_influxdb/v1.9/troubleshooting/frequently-asked-questions/enterprise_influxdb/v1.9/administration/config-data-nodes/
```

Comment:
something amiss with the double versions here--let's look at this one, is the context attempting to land on the Troubleshooting page from v1.3 or earlier?